ajax/libs/hyperform/package.json

```diff
@@ -0,0 +1,35 @@
+{
+  "name": "hyperform",
+  "filename": "hyperform.min.js",
+  "version": "0.8.10",
```

Comment:
Please remove `version` info so that you can add a lib with a single `package.json`, thanks.